signac/contrib/collection.py

```diff
@@ -120,12 +151,47 @@ def __delitem__(self, key):
                          _float(key) if type(key) is float else key)
 
     def get(self, key, default=None):
+        """Get the value for given key.
+
+        Parameters
+        ----------
+        key :
```

Comment:
missing key argument type (I'm assuming its a string)

Reply:
@lyrivera actually there is check in the same method to see if key is a float. So I think it can have float as well as str data type.